apps/messaging/app/[locale]/pre-login/route.tsx

```diff
@@ -35,7 +39,7 @@ export async function GET(request: NextRequest) {
 
   cookies().set(socialConnectorIdCookieName, connectorId, {
     secure: isLocal,
-    domain: isLocal ? undefined : ".services.gov.ie",
+    domain: process.env.COOKIE_DOMAIN,
```

Comment:
```suggestion
    domain: process.env.COOKIE_DOMAIN ?? undefined,
```
To avoid to set empty string, default value must be undefined

Reply:
How do you set empty string in .env? Blank space?